web/client/actions/geostory.js

```diff
@@ -172,3 +173,22 @@ export const move = (source, target, position) => ({
     target,
     position
 });
+
+/**
+ * Toggle focus on content
+ * @param {boolean} status if true set the focus otherwise remove
+ * @param {object} target the target content to be focused
+ * @param {object} section the story section parent of the target
+ * @param {object} parents all the target's ancestor, parent section excluded
+ * @param {boolean} isBackground true if the target is the background of the target
```

Comment:
instead of `isBackground` you could add a custom selector or selector tail parameter. This should simplify and generalize the code, to create the proper selector in "withFocusMask"
instead isBackground in immersive can be replaced with hideContents, as an option.

I should re-write the action like this

`setFocusOnContent(status, target, selector, options)`